is-with-analytics/is/identity-server-deployment.yaml

```diff
@@ -90,7 +90,7 @@ spec:
           name: identity-server-conf-event-publishers
       - name: shared-deployment-persistent-disk
         persistentVolumeClaim:
-          claimName: scalable-is-shared-deployment-volume-claim
+          claimName: is-shared-deployment-volume-claim
```

Comment:
shale we make this `identity-shared-deployment-volume-claim`, if not it looks like a boolean variable.

Reply:
+1, we will make this change for next release.

Reply:
Yes, please create a issue to track this. We need to fix the across the repo.